types/three/src/core/EventDispatcher.d.ts

```diff
@@ -13,12 +13,31 @@ export type EventListener<E, T, U> = (event: E & { type: T } & { target: U }) =>
 
 /**
  * JavaScript events for custom objects
- *
- * @source src/core/EventDispatcher.js
+ * @example
+ * ```typescript
+ * // Adding events to a custom object
+ * class Car extends EventDispatcher {
+ *   start() {
+ *     this.dispatchEvent( { type: 'start', message: 'vroom vroom!' } );
+ *   }
+ * };
+ * // Using events with the custom object
+ * const car = new Car();
+ * car.addEventListener( 'start', ( event ) => {
+ *   alert( event.message );
+ * } );
+ * car.start();
+ * ```
+ * @see {@link https://github.com/mrdoob/eventdispatcher.js | mrdoob EventDispatcher on GitHub}
+ * @see {@link https://threejs.org/docs/index.html#api/en/core/EventDispatcher | Official Documentation}
+ * @see {@link https://github.com/mrdoob/three.js/blob/master/src/core/EventDispatcher.js | Source}
  */
 export class EventDispatcher<E extends BaseEvent = Event> {
     /**
-     * Creates eventDispatcher object. It needs to be call with '.call' to add the functionality to an object.
+     * Creates {@link THREE.EventDispatcher | EventDispatcher} object.
+     * @remarks
+     * It needs to be call with {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call | Function.prototype.call()}
+     * to add the functionality to an object.
```

Comment:
Is this a remark that is still active? I feel like it's a stale description since we are using classes now

Reply:
The Implementation Internally still uses .call. But I almost removed this doc from the old version and decided to keep it. But no problem I'm happy removing too. Fixed.

Reply:
I see! I think it's now a deprecated use even internally so it's fine to remove from docs side